This file is part of MXE. See LICENSE.md for licensing information.

From a9c84ba7518ea4cc92cf4b391f609c3f6ade0974 Mon Sep 17 00:00:00 2001
From: Boris Nagaev <bnagaev@gmail.com>
Date: Sun, 30 Aug 2015 16:13:06 +0200
Subject: [PATCH] fix logging

Set logging function to simple fprintf.
---
 src/c_init.cpp |    3 ++-
 src/console.h  |    3 +--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/c_init.cpp b/src/c_init.cpp
index 542b066..a1ce500 100644
--- a/src/c_init.cpp
+++ b/src/c_init.cpp
@@ -141,7 +141,7 @@ console_t console_init =
     intro
 };
 
-
+/*
 void __acc_cdecl_va con_fprintf(FILE *f, const char *format, ...)
 {
     va_list args;
@@ -156,6 +156,7 @@ void __acc_cdecl_va con_fprintf(FILE *f, const char *format, ...)
     assert(con != me);
     con->print0(f,buf);
 }
+*/
 
 #endif /* USE_CONSOLE */
 
diff --git a/src/console.h b/src/console.h
index eeee36e..4a2c4c0 100644
--- a/src/console.h
+++ b/src/console.h
@@ -123,8 +123,7 @@ console_t;
 
 
 #if defined(__GNUC__)
-void __acc_cdecl_va con_fprintf(FILE *f, const char *format, ...)
-        __attribute__((__format__(printf,2,3)));
+#define con_fprintf fprintf
 #else
 void __acc_cdecl_va con_fprintf(FILE *f, const char *format, ...);
 #endif
-- 
1.7.10.4

